backend/src/main/java/gov/cdc/usds/simplereport/service/PersonService.java

```diff
@@ -94,11 +107,19 @@ public String addPatient(
 			residentCongregateSetting,
 			employedInHealthcare
 		);
+
+		if (newPatient.getRole() != PersonRole.STAFF) {
```

Comment:
Our fronted doesn't have a metaphor for not associating a person to a facility. It will be equally out of place on frontend as it is here. We should table this until we figure out roles and permissions